docs/guides/build-composition.md

```diff
@@ -0,0 +1,279 @@
+---
+tocHidden: true
+---
+
+# Build your Composition

```

Comment:
Same as the build-xrd.md file, move this to the `title` front matter instead of a top-level header (`#`)